pyproject.toml

```
@@ -12,7 +12,7 @@ classifiers = [
 ]
 version = "0.4.6"
 dependencies = ["pytest >= 3.6", "freezegun >= 1.0"]
-requires-python = ">= 3.6"
+requires-python = ">= 3.7"
```

```suggestion
requires-python = ">= 3.6"
```

Let's not add restriction in the package metadata unless there's actually a problem with the code on 3.6. It just creates needless churn for dependencies.

---

Then I suggest keeping support for it and testing it on the CI: it's arguably worse if it suddenly fails for users and we missed it during testing.

---

The CI only runs on changes, so we will miss it anyway. But if you want to add some actions which regularly test (weekly schedule or something?) then it can make sense to add 3.6 back.

---

Yes, what I meant is there may be a change that breaks compatibility for 3.6.

A PR is opened, the tests run on 3.7, 3.11 and 3.12-dev, and pass.

We don't detect it broke on 3.6, so merge it, and release it.

Because of `requires-python = ">= 3.6"` it installs for end users on 3.6, but they find out it won't run.

---

https://github.com/pytest-dev/pytest-freezer/commit/0fffe43f506799207c9742564b85bb7eaf254a19